editor/editor_resource_picker.cpp

```diff
@@ -602,32 +632,28 @@ bool EditorResourcePicker::_is_drop_valid(const Dictionary &p_drag_data) const {
 		}
 	} else if (drag_data.has("type") && String(drag_data["type"]) == "resource") {
 		res = drag_data["resource"];
+	} else if (drag_data.has("type") && String(drag_data["type"]) == "files") {
+		Vector<String> files = drag_data["files"];
+
+		if (files.size() == 1) {
+			String file = files[0];
+			res = ResourceLoader::load(file);
```

Comment:
This can be problematic and, more importantly, slow. But I guess it's the only way to detect the custom types for now? Ideally, this is where you want to access EditorFileSystem's cache, directly, without going through the new `_get_resource_type`. Which is exactly what the old code did.

Reply:
I'm guessing this is what causes all the problems and the need for core changes in the QuickOpen PR?

Reply:
Yes, exactly.